dotnet/src/SemanticKernel.Abstractions/Contents/FunctionResultContent.cs

```diff
@@ -16,7 +16,7 @@ public sealed class FunctionResultContent : KernelContent
     /// The function call ID.
     /// </summary>
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
-    public string? Id { get; }
+    public string? CallId { get; }
```

Comment:
Why do we need this change?

We should be avoiding breaking changes.

Reply:
This was decided in the ADR discussion regarding the confusion on this being a `Call` or a `Result` identifier.